xscen/ensembles.py

```diff
@@ -441,7 +475,85 @@ def generate_weights(
         # Drop the experiment coordinate
         weights = weights.drop_vars("experiment")
 
+    # Attribute_weights
+    if attribute_weights:
+        stationary_weights = {}
+        non_stationary_weights = {}
+        for att, v_att in attribute_weights.items():
+            # Add warning when mismatch between independance_level/experiment_weight and attribute_weights
+            if att != independence_level or (
```

Comment:
This will only catch `institution`, but not `driving_model/GCM`. We also don't want to warn for attributes that do not matter here (e.g. `mip_era`).

I would be more explicit.